validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -800,7 +800,7 @@ protected void resetConfig(boolean fullReset) {
       resetDeviceConfig(false);
       updateConfig("soft reset");
       debug("Done with reset_config");
-      resetRequired = false;
+      // resetRequired = false;
```

Comment:
?

Reply:
I just changed field to non-static. The previous test config doesn't seem to be cleaned up correctly.

I am getting discovery config on next test if I use more then 3 test class.

`sequencer broken_config periodic_scan too_much_state`